src/Tempest/Support/src/PathHelper.php

```diff
@@ -4,8 +4,67 @@
 
 namespace Tempest\Support;
 
+use Exception;
+use Tempest\Core\Composer;
+use Tempest\Core\Kernel;
+use function Tempest\get;
+
 final readonly class PathHelper
 {
+    public static function root(string ...$paths): string
```

Comment:
I don't think we need this one anymore. Use `root_path()` instead

Reply:
You're right, i'll correct this ASAP